core-bundle/contao/models/PageModel.php

```diff
@@ -708,7 +708,7 @@ public static function findPublishedFallbackByHostname($strHost, array $arrOptio
 		}
 
 		$t = static::$strTable;
-		$arrColumns = array("$t.dns=? AND $t.fallback=1");
+		$arrColumns = array("$t.type='root' AND $t.dns=? AND $t.fallback=1");
```

Comment:
Same adjustment needs to be done 4 lines below.

Reply:
Fixed.